package.json

```diff
@@ -100,11 +100,12 @@
         "mocha-webpack": "^2.0.0-beta.0",
         "nyc": "^12.0.2",
         "seedrandom": "^2.4.3",
+        "sharp": "^0.20.3",
         "shx": "0.2.2",
         "sinon": "^4.2.2",
         "ts-loader": "^4.3.1",
         "tslint": "^5.10.0",
-        "typescript": "~2.7.1",
+        "typescript": "^2.8",
```

Comment:
Description and package-lock.json say 2.9. I'm wondering if Angular (6, CLI) might have problems with Typescript > 2.8?

Reply:
Shouldn't as the final bundle comes in JS with only definition files. But we should test it anyway, that's a good point ~as JS works as it wants to~.

Reply:
Besides that it looks fine to me I really like and appreciate the typed exceptions!! Great work